examples/install/istio/helm/README.md

```diff
@@ -15,8 +15,8 @@ helm repo update
 ## Environment Variables
 ```sh
 ISTIO_VERSION=1.12.6
-REMOTE_CLUSTER1=cluster1
-REMOTE_CLUSTER2=cluster2
+REMOTE_CLUSTER1=cluster-1
```

Comment:
I was trying to keep it consistent with our official quickstart docs.  https://docs.solo.io/gloo-mesh-enterprise/latest/getting_started/managed_kubernetes/  I thought the local install was the same but its also different.  This should probably be left alone for now.